css/style.css

```diff
@@ -0,0 +1,115 @@
+/*style.css*/
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+.clearfix:after {
+     content: ".";
+     clear: both;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px;
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0;
+}
+
+.wrapper {
+  width: 100%;
+  margin: 0px auto;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  font-size: 12px;
```

Comment:
Now that we're through the responsive week - try changing your font sizes to use either `em`s or `rem`s so that your fonts scale easily on phones and tablets.